packages/react-components/components/select/src/select.jsx

```diff
@@ -32,7 +36,8 @@ const propTypes = {
 };
 
 const defaultProps = {
-    inline: false
+    inline: false,
+    multiple: false
```

Comment:
You dont need to add `multiple` here since you don't use it, it would be provided to the select component through `...rest`.